### PR TITLE
Implement Capistrano

### PR DESCRIPTION
**Pull request by bsp3ars**
Adds capistrano support to the treatment database application. Also implements a systems config file for monitoring puma

### CODE REVIEW

**Comment by bsp3ars** [comment]
In order to get the CircleCI build to pass, we will need to change the RAILS_MASTER_KEY in CircleCI in Project Settings > Environment Variables.

**Comment by hortongn** [comment]
@bsp3ars The CircleCI problem is fixed.  CI was running the app in "test" mode like it should, but because the RAILS_MASTER_KEY variable had been added to the project settings the app was trying to use the (old) key even though it didn't need it.  So I just had to delete the master key from the CircleCI settings and the job then passed.

Can you please squash you commits and push up again?  I'll then review and merge.

File: config/deploy/production.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+set :rails_env, :production
+set :bundle_without, %w[development test].join(' ')
+set :branch, 'main'
+set :default_env, path: '$PATH:/usr/local/bin'
+set :bundle_path, -> { shared_path.join('vendor/bundle') }
+append :linked_dirs, 'tmp', 'log'
+ask(:username, nil)
+ask(:password, nil, echo: false)
+server 'libapps2.libraries.uc.edu', user: fetch(:username), password: fetch(:password), port: 22, roles: %i[web app db]
```
Comment (hortongn) [inline config/deploy/production.rb:11]: Change `libapps2.libraries.uc.edu` to `libapps.libraries.uc.edu`

File: Gemfile [hunk 2]
```diff
@@ -72,6 +69,10 @@ gem 'brakeman', '~> 5.1', '>= 5.1.1'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'capistrano', '3.11.2'
```
Comment (hortongn) [inline Gemfile:72]: Do we really need to be pinned to capistrano v3.11.2?

**Review by hortongn — CHANGES_REQUESTED** [review]
@bsp3ars See my comments inline.  Also please squash your commits.  Thanks

**Review by hortongn — APPROVED** [review]
@bsp3ars Well done.  Merging

**Pull request merged**